<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingInterface</class>
 <widget class="QWidget" name="settingInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QWidget" name="bgWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>50</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QWidget" name="importWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="SubtitleLabel" name="SubtitleLabel">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>TTS 引擎设置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CardWidget" name="CardWidget">
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <property name="leftMargin">
             <number>20</number>
            </property>
            <property name="topMargin">
             <number>20</number>
            </property>
            <property name="rightMargin">
             <number>20</number>
            </property>
            <property name="bottomMargin">
             <number>20</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="enginePathLabel">
                <property name="styleSheet">
                 <string notr="true">font: 700 10pt 'Segoe UI', 'Microsoft YaHei', 'PingFang SC';</string>
                </property>
                <property name="text">
                 <string>引擎选择</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="CaptionLabel" name="engineCaptionLabel">
                <property name="text">
                 <string>选择可用的 TTS 引擎</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="ComboBox" name="engineComboBox">
              <property name="minimumSize">
               <size>
                <width>240</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>240</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="CardWidget" name="CardWidget_2">
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <property name="leftMargin">
             <number>20</number>
            </property>
            <property name="topMargin">
             <number>20</number>
            </property>
            <property name="rightMargin">
             <number>20</number>
            </property>
            <property name="bottomMargin">
             <number>20</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="ratePathLabel">
                <property name="styleSheet">
                 <string notr="true">font: 700 10pt 'Segoe UI', 'Microsoft YaHei', 'PingFang SC';</string>
                </property>
                <property name="text">
                 <string>语速设置</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="CaptionLabel" name="rateCaptionLabel">
                <property name="text">
                 <string>调节发声速度，默认为 200</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="SpinBox" name="rateSpinBox">
              <property name="minimumSize">
               <size>
                <width>180</width>
                <height>33</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>180</width>
                <height>33</height>
               </size>
              </property>
              <property name="maximum">
               <number>500</number>
              </property>
              <property name="value">
               <number>200</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="CardWidget" name="CardWidget_3">
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <property name="leftMargin">
             <number>20</number>
            </property>
            <property name="topMargin">
             <number>20</number>
            </property>
            <property name="rightMargin">
             <number>20</number>
            </property>
            <property name="bottomMargin">
             <number>20</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_10">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="volumePathLabel">
                <property name="styleSheet">
                 <string notr="true">font: 700 10pt 'Segoe UI', 'Microsoft YaHei', 'PingFang SC';</string>
                </property>
                <property name="text">
                 <string>音量设置</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="CaptionLabel" name="volumeCaptionLabel">
                <property name="text">
                 <string>调节音量，默认为 1.0</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="DoubleSpinBox" name="volumeDoubleSpinBox">
              <property name="minimumSize">
               <size>
                <width>180</width>
                <height>33</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>180</width>
                <height>33</height>
               </size>
              </property>
              <property name="maximum">
               <double>1.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="CardWidget" name="CardWidget_4">
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="savePathLabel">
             <property name="styleSheet">
              <string notr="true">font: 700 10pt 'Segoe UI', 'Microsoft YaHei', 'PingFang SC';</string>
             </property>
             <property name="text">
              <string>保存设置</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="CaptionLabel" name="saveCaptionLabel">
             <property name="text">
              <string>保存后，需要重新导入文件才能生效</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="PushButton" name="savePushButton">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>180</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>保存设置</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="SubtitleLabel" name="SubtitleLabel_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>关于</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CardWidget" name="importCardWidget_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>80</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>20</number>
           </property>
           <property name="leftMargin">
            <number>20</number>
           </property>
           <property name="topMargin">
            <number>20</number>
           </property>
           <property name="rightMargin">
            <number>20</number>
           </property>
           <property name="bottomMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="IconWidget" name="versionIconWidget">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="versionShowLabel">
               <property name="styleSheet">
                <string notr="true">font: 700 9pt 'Segoe UI', 'Microsoft YaHei', 'PingFang SC';</string>
               </property>
               <property name="text">
                <string>当前版本</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="CaptionLabel" name="versionLabel">
               <property name="text">
                <string>0.0.0</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="PrimaryPushButton" name="versionPrimaryPushButton">
             <property name="minimumSize">
              <size>
               <width>180</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>180</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>检查更新</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="CardWidget" name="importCardWidget">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>80</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>20</number>
           </property>
           <property name="leftMargin">
            <number>20</number>
           </property>
           <property name="topMargin">
            <number>20</number>
           </property>
           <property name="rightMargin">
            <number>20</number>
           </property>
           <property name="bottomMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="IconWidget" name="copyrightIconWidget">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="copyrightShowLabel">
               <property name="styleSheet">
                <string notr="true">font: 700 9pt 'Segoe UI', 'Microsoft YaHei', 'PingFang SC';</string>
               </property>
               <property name="text">
                <string>版权</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="CaptionLabel" name="copyrightLabel">
               <property name="text">
                <string>Copyright © 2023 Nagisa</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="TransparentToolButton" name="githubButton"/>
           </item>
           <item>
            <widget class="TransparentToolButton" name="giteeButton"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ToolButton</class>
   <extends>QToolButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TransparentToolButton</class>
   <extends>ToolButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>IconWidget</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CaptionLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SubtitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SpinBox</class>
   <extends>QSpinBox</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>DoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
